cmd/zed/serve/command.go

```diff
@@ -59,6 +64,7 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	})
 	f.StringVar(&c.conf.DefaultResponseFormat, "defaultfmt", service.DefaultZedFormat, "default response format")
 	f.StringVar(&c.listenAddr, "l", ":9867", "[addr]:port to listen on")
+	f.DurationVar(&c.manage, "manage", 0, "manage lake (e.g., compact) every interval (default 0=never)")
```

Comment:
The usage message shows the default twice.
```console
$ go run ./cmd/zed serve -h |& grep manage
    -manage manage lake (e.g., compact) every interval (default 0=never) (default "0s")
```

```suggestion
	f.DurationVar(&c.manage, "manage", 0, "when positive, run lake maintenance tasks at this interval")
```